src/diffusers/utils/export_utils.py

```diff
@@ -37,7 +37,7 @@ def export_to_gif(image: List[PIL.Image.Image], output_gif_path: str = None) ->
         save_all=True,
         append_images=image[1:],
         optimize=False,
-        duration=100,
+        duration=1000 // fps,
```

Comment:
Thank you for reviewing.

Since `duration` can specify the number of seconds in ms to display one frame, I think it will work correctly at `1000 // fps`.

I attached gifs I actually tried my code at fps 1 and 10. I generated 16 frames by PIA example code.

1 fps -> 1000 duration
![Pia Animation 1fps](https://github.com/huggingface/diffusers/assets/1396267/4f783abe-7c26-4f08-aa6b-e0581f9e742f)


10 fps -> 100 duration
![Pia Animation 10fps](https://github.com/huggingface/diffusers/assets/1396267/dc400898-3b7d-4c4c-baef-aecf1f94276e)

Reply:
Ah okay. Misunderstood how duration was applied. Looks good to me 👍🏽